src/Plugin/EntityBrowser/SelectionDisplay/NewDisplay.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/**
+ * Contains \Drupal\entity_browser\Plugin\EntityBrowser\SelectionDisplay\NewDisplay.
+ */
+
+namespace Drupal\entity_browser\Plugin\EntityBrowser\SelectionDisplay;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Component\Utility\NestedArray;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\entity_browser\SelectionDisplayBase;
+
+/**
+ * Show current selection and delivers selected entities.
+ *
+ * @EntityBrowserSelectionDisplay(
+ *   id = "new_display",
+ *   label = @Translation("New selection display"),
+ *   description = @Translation("Show current selection display and delivers selected entities.")
+ * )
+ */
+class NewDisplay extends SelectionDisplayBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return array(
+      'view' => NULL,
+      'view_display' => NULL,
+    ) + parent::defaultConfiguration();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getForm(array &$original_form, FormStateInterface $form_state) {
+    $form = [];
+    $form['#attached']['library'][] = 'entity_browser/new_display';
+
+    $selected_entities = $form_state->get(['entity_browser', 'selected_entities']);
+    $ids = array_keys($selected_entities);
+
+    $hidden_id = Html::getUniqueId('edit-selected-target-id');
+    $details_id = Html::getUniqueId('edit-selected');
+
+    $form += [
+      '#id' => $details_id,
+      '#type' => 'details',
+      '#open' => !empty($ids),
+      'target_id' => [
```

Comment:
This was copied from entity reference widget and it makes no sense here. 

We use this fields there to store IDs of referenced entities that field needs. We don't have anything like that here.